application/common/mail/mfa-option-added.text.php

```diff
@@ -21,8 +21,23 @@
  * @var string $supportEmail
  * @var string $supportName
  * @var bool   $isMfaEnabled
+ * @var array  $mfaOptions
  */
 ?>
 Dear <?= $displayName ?>,
 
-TODO: Compose contents of this email message.
\ No newline at end of file
+You have added a new option for 2-Step Verification to your <?= $idpDisplayName ?> Identity account.
+When you are prompted for 2-Step Verification in the future, you will now have the option to choose a different
+method of 2-Step Verification if you do not have access to the primary method.
+
+You now have the following 2-Step Verification options enabled:
+<?php
+    foreach ($mfaOptions as $mfa) {
+       echo ' - ' . $mfa->getReadableType();
```

Comment:
You'll probably want to add a newline (`\n`) to the end of this to force them each onto their own line.